opba-protocols/xs2a-protocol/src/test/java/de/adorsys/opba/protocol/xs2a/config/MapperTestConfig.java

```diff
@@ -0,0 +1,28 @@
+package de.adorsys.opba.protocol.xs2a.config;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import de.adorsys.opba.protocol.xs2a.domain.dto.forms.ScaMethod;
+import org.mapstruct.factory.Mappers;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan({"de.adorsys.opba.protocol.xs2a.service.mappers.generated",
+        "de.adorsys.opba.protocol.xs2a.util",
+        "de.adorsys.opba.protocol.xs2a.domain.dto.forms"})
+public class MapperTestConfig {
+    @Bean
+    public ObjectMapper objectMapper() {
+        return new ObjectMapper()
+                       .registerModule(new JavaTimeModule())
+                       .enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+    }
+
+    @Bean
+    public ScaMethod.FromAuthObject fromAuthObject() {
```

Comment:
We should annotate `FromAuthObject` as `@Mapper(componentModel = SPRING_KEYWORD,...)` and inject it instead of using Mapper.getMapper

Reply:
@Bean removed, Mapper was made a spring bean